contrib/initramfs/hooks/zfsunlock.in

```diff
@@ -15,4 +15,5 @@ esac
 
 . /usr/share/initramfs-tools/hook-functions
 
+mkdir -p "$DESTDIR/usr/bin/"
 copy_exec /usr/share/initramfs-tools/zfsunlock /usr/bin
```

Comment:
copy_exec calls copy_file, which has a mkdir in it. Instead of adding a mkdir, does this work:
`copy_exec /usr/share/initramfs-tools/zfsunlock /usr/bin/zfsunlock`

Reply:
I agree, this would be better.